src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -144,8 +147,17 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
     val promise = Promise[Seq[Try[Unit]]]()
 
     Future.sequence(batchStatements).onComplete {
-      case Success(_) => promise.complete(Success(Nil)) // Nil == all good
-      case Failure(e) => promise.failure(e)
+      case Success(_) =>
+        for (
+          p <- pubsub;
+          tag: String <- serialized.map(_.payload.map(_.tags).flatten).flatten.toSet
+        ) {
+          p ! DistributedPubSubMediator.Publish("akka.persistence.cassandra.journal.tag", tag)
```

Comment:
this can become rather many messages at a high frequence that are sent out, I think we need some throttling. I don't want to pull in akka-contrib, otherwise this could work http://doc.akka.io/docs/akka/2.4.1/contrib/throttle.html
I think we should just add something simple in a child actor.